packages/client/src/lib/interface.ts

```diff
@@ -1,7 +1,8 @@
 import type { CID } from 'multiformats'
 export type { CID }
 
-import type { CarReader } from '@ipld/car'
+import type { RootsReader, BlockReader } from '@ipld/car/api'
+type CarReader = RootsReader & BlockReader
```

Comment:
Paging @rvagg. Perhaps for convenience @ipld/car should export an interface for CarReaderish things